### ISSUE TITLE
Escalating areas with large number of linting issues

### ISSUE THREAD

**Issue by planemad** [osmlab/osmlint#188]
A common complaint with to-fix is that one does not see the larger picture of what is being fixed. This is a problem when there is bad changeset with multiple errors that is better to be reverted than fixed via to-fix.

<img width="469" alt="screenshot 2016-12-21 18 25 14" src="https://cloud.githubusercontent.com/assets/126868/21390066/df176274-c7aa-11e6-8bb1-fcfe71e1fbe1.png">

An example is when [this accidental changeset](http://osmcha.mapbox.com/44196753/) introduced a large number of connectivity errors throughout a city. When the original mapper and DWG initiated a cleanup by reverting the changesets it ran into conflicts because of fixes that happened to the objects via to-fix.

@Rub21 @geohacker Fixing these issues via to-fix made the data harder to cleanup, how can we better handle issues like this to avoid such a scenario? Could we flag this separately somehow?

cc @manoharuss

**Comment by Rub21** [comment]
@planemad 
> This is a problem when there is bad changeset with multiple errors that is better to be reverted than fixed fixed by micro tasking.


I completely agree with that, We are working a lot with to-fix fixing many issues per day. from now on we are going report or revert this changest in osmcha. 

right now something automatic which we can do in to-fix  is not possible, i think the only way so far is to check the changeset and set as harmful or not or just an unintentional edition


cc. @calfarome
